.github/ISSUE_TEMPLATE/community-call-takeaways.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Community Call Takeaways
+about: Use this issue to request the creation of a document on Community Call takeaways
+title: Community Call Takeaways
+labels: good first issue, events
+assignees: ''
+
+---
+
+Purpose & Details
+Create a shareable list of insights from our last Community Call discussion. These insights will be used to create content for our social channels.
+
+Acceptance Criteria
+ - [ ] We have checked the Marketing Slack channel for the community call anonymised discussion document.
+ - [ ] We have produced an article detailing the takeaways from the community call in Google Drive.
+ - [ ] We have shared the draft in the Marketing channel on Slack asking people to contribute/ provide feedback on the document
+ - [ ] We have implemented the feedback
+ - [ ] We have created an article on LinkedIn and shared it with InnerSource Commons 
+        - [ ] Please share the article on Linkedin and tag InnerSource Commons
+        - [ ] Please share a link to the article in the Marketing Slack channel
```

Comment:
Are these two bullets meant to be indented?

Reply:
Yes, I saw them as sub-bullets to the bullet point on top but if you think they are ok as stand-alone I can remove the indentation